modules/Microsoft.Synapse/privateLinkHubs/.test/common/dependencies.bicep

```
@@ -0,0 +1,73 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Network Security Group to create.')
+param networkSecurityGroupName string
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+resource networkSecurityGroup 'Microsoft.Network/networkSecurityGroups@2021-08-01' = {
+  name: networkSecurityGroupName
+  location: location
+  properties: {
+  }
+}
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+    name: virtualNetworkName
+    location: location
+    properties: {
+        addressSpace: {
+            addressPrefixes: [
+                '10.0.0.0/16'
+            ]
+        }
+        subnets: [
+            {
+                name: 'defaultSubnet'
+                properties: {
+                    addressPrefix: '10.0.6.0/24'
+                    networkSecurityGroup: {
+                        id: networkSecurityGroup.id
+                    }
+                    privateEndpointNetworkPolicies: 'Disabled'
+                    privateLinkServiceNetworkPolicies: 'Enabled'
+                }
+            }
+        ]
+    }
+}
+
+resource privateDNSZone 'Microsoft.Network/privateDnsZones@2020-06-01' = {
+    name:  'privatelink.azuresynapse.net'
+    location: 'global'
+
+    resource virtualNetworkLinks 'virtualNetworkLinks@2020-06-01' = {
+        name: '${split( 'privatelink.azuresynapse.net', '.')[0]}-vnet-link'
```

To align with other module tests please replace with
```suggestion
        name: '${virtualNetwork.name}-vnetlink'
```